Tests/ExportersTests/OpenTelemetryProtocol/OtlpMetricExporterTests.swift

```diff
@@ -37,6 +42,42 @@ class OtlpMetricExproterTests: XCTestCase {
         exporter.shutdown()
     }
 
+    func testImplicitGrpcLoggingConfig() throws {
+        let exporter = OtlpMetricExporter(channel: channel)
+        guard let logger = exporter.callOptions?.logger else {
+            throw "Missing logger"
```

Comment:
Is there something I need to do to prevent Codecov from checking for coverage within test code?

Reply:
Answered my own question - added a `.codecove.yml` file that excludes the `Tests` folder.